about.html

```diff
@@ -0,0 +1,137 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <title >About me.</title>
+  </head>
+  <body class="full-body" >
+    <img src="./images/Union.png" alt="hamburger-button"  class="hamburger" onclick="openpopup()">
+
+    <div class="popup" id="popup">
+        <img src="./images/cloasepopup.svg" alt="close button" class="closepopup" onclick="closepopup()">
+        <ul>
+            <li><a href="about.html">About</a></li> 
+            <li><a href="index.html">Home</a></li>
+            <li><a href="#form">Join</a></li>
+            <li><a href="#contact">Contact us</a></li>
+        </ul>
+       
+    </div>
+
+    <a href="./index.html"><img src="./images/240957a3-789c-4e2b-9fc7-34a08b3e9f0c.png" class="crypto-image" alt="crypto-logo" ></a>
+    <nav class="navigation"> 
+       
+        <div class="nav-text">
+        <div class="about"> <a href="./about.html"> About</a></div>
+        <div class="home"> <a href="./index.html">Home</a></div>
+        <div class="join"><a href="#form">Join</a></div>
+        <div class="contact" > <a href="#contact">Contact us </a></div>
+        </div>
+    </nav>
+
+    <section class="intro-section">
+      <div class="crypto-text">"Blockchain crypto!"</div>
+      
+      <h1 class="about-heading">NEW CRYPTO COIN 2022 INITIAL COIN OFFERING</h1>
+      <hr class="speaker-line1" />
+      <p class="crypto-words">
+        The crypto coin brings together investors ,financial institutions
+        governments and regular individuals to all have an impact in improving
+        the infrastructure and and increasing the level of technology used
+        across all sectors of life.This will be seen through the development of
+        the ecosystem which will me carried out by Artificial intelligence that
+        determines where to invest and develop in order to bring the highest
+        returns for the project.
+      </p>
```

Comment:
Your p tag text styling is correctly implemented in the Homepage Design. But is expanded from right to left on the about.html page. Kindly change it like you did in index.html.
| Index.html Design   | about.html Design  |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/61361037/219809676-13e50993-fdcc-4ddc-bf42-48eb930b8327.png)| ![image](https://user-images.githubusercontent.com/61361037/219809516-f8dc6be1-6d1f-4f70-a37b-9b99b82e7042.png)|